tempto-core/src/test/groovy/com/teradata/tempto/assertions/QueryAssertTest.groovy

```diff
@@ -56,6 +57,11 @@ public class QueryAssertTest
     }
   }
 
+  def setupSpec()
+  {
+    System.setProperty(TEST_CONFIGURATION_URI_KEY, "/configuration/global-configuration-tempto.yaml");
```

Comment:
Isn't there any better way than introducing config file to tests?
Cannot we just decouple unit tests from config file?